aws/structure.go

```diff
@@ -1525,6 +1525,41 @@ func flattenLambdaEnvironment(lambdaEnv *lambda.EnvironmentResponse) []interface
 	return []interface{}{envs}
 }
 
+func expandLambdaEventSourceMappingDestinationConfig(vDest []interface{}) *lambda.DestinationConfig {
```

Comment:
This function can immediately `return nil` if there are no elements in the list (which will allow it to always be called regardless of SQS or not), e.g.

```go
if len(vDest) == 0 {
	return nil
}
```